doc/release-notes-15437.md

```diff
@@ -32,3 +32,21 @@ Please use the recommended alternatives if you rely on this deprecated feature:
   could wait until the transaction has confirmed (taking into account the fee
   target they set (compare the RPC `estimatesmartfee`)) or listen for the
   transaction announcement by other network peers to check for propagation.
+
+The removal of BIP61 REJECT message support also has the following minor RPC
+and logging implications:
+
+* `testmempoolaccept` and `sendrawtransaction` no longer return the P2P REJECT
+  code when a transaction is not accepted to the mempool. They still return the
+  verbal reject reason.
+
+* Log messages that previously reported the REJECT reason when a transaction was
```

Comment:
Not sure about this statement, at least the reject-reason should be logged for the broadcastTransaction call in SubmitMemoryPoolAndRelay, which ones were you thinking off ?

Reply:
Sorry, this should say REJECT code, not REJECT reason. I'll update the docs.